src/hexdoc/_templates/index.css.jinja

```diff
@@ -154,6 +154,105 @@ blockquote.crafting-info {
   position: absolute;
 }
 
+.item-slot {
+  position: relative;
+  width: min-content;
+}
+
+.item-slot > .texture {
+  position: absolute;
+  top: 10px;
+  left: 10px;
+}
+
+.extra-info .texture{
+  width: 16px;
+  height: 16px;
+  margin: 2px;
+}
+
+.extra-info{
+  display: grid;
+  grid-auto-flow: column;
+  align-items: center;
+  justify-content: start;
+}
+
+.smelting-recipe{
+  position:relative;
+}
+
+.smelting-input{
+  position: absolute;
+  top: 50px;
+  left: 54px;
+}
+
+.smelting-result{
+  position: absolute;
```

Comment:
could move this part into a common `.recipe-result` class

Reply:
the issue with all these is that you'd still need them separate to handle the spacing too. Unless we moved spacing stuff into inline jinja. Up to you though, but this mess is why I suggested it earlier.

Reply:
right, but the `position: absolute;` doesn't need to be duplicated. every result has it, so add the common class to every result